core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/scan/FileDecodingMethod.java

```diff
@@ -18,15 +16,9 @@ public enum FileDecodingMethod {
         this.charset = charset;
     }
 
-    // use the name string instead of enum string in JSON
-    @JsonValue
-    public String getName() {
-        return this.name;
-    }
-
     @Override
     public String toString() {
-        return this.getName();
+        return this.name;
     }
```

Comment:
it might still be valuable to keep the getter for serializers to use.

Reply:
Fixed.